src/openapi.yaml

```diff
@@ -133,6 +133,12 @@ paths:
                         name:
                           type: string
                           description: User's Name
+          '401':
```

Comment:
@jehrlich Note, I applied 401s to all endpoints -- I assumed this was valid and matched your comment in Slack. If not, please advise...